README.md

```diff
@@ -257,6 +259,25 @@ Default port for `http` endpoints is __9000__ if the port is left unspecified.
 
 Alternatively the endpoint can also be specified by setting an `AWS_ENDPOINT` environment variable.
 
+### Custom Engine
```

Comment:
Great job adding this explanation to the readme

Reply:
Thanks! 😄